BUILD.md

```diff
@@ -386,85 +376,35 @@ For example:
 
     make -j4
 
-You can now run the demo applications from the command line:
-
-    open demos/cube.app
-    open demos/cubepp.app
-    open demos/smoketest.app
-    open demos/vulkaninfo.app
-
-Or you can locate them from `Finder` and launch them from there.
-
-##### The Install Target and RPATH
-
-The applications you just built are "bundled applications", but the executables
-are using the `RPATH` mechanism to locate runtime dependencies that are still
-in your build tree.
-
-To see this, run this command from your `build` directory:
-
-    otool -l demos/cube.app/Contents/MacOS/cube
-
-and note that the `cube` executable contains loader commands:
-
-- `LC_LOAD_DYLIB` to load `libvulkan.1.dylib` via an `@rpath`
-- `LC_RPATH` that contains an absolute path to the build location of the Vulkan loader
-
-This makes the bundled application "non-transportable", meaning that it won't run
-unless the Vulkan loader is on that specific absolute path.
-This is useful for debugging the loader or other components built in this repository,
-but not if you want to move the application to another machine or remove your build tree.
-
-To address this problem, run:
-
-    make install
-
-This step "cleans up" the `RPATH` to remove any external references
-and performs other bundle fix-ups.
-After running `make install`, re-run the `otool` command again and note:
-
-- `LC_LOAD_DYLIB` is now `@executable_path/../MacOS/libvulkan.1.dylib`
-- `LC_RPATH` is no longer present
-
-The "bundle fix-up" operation also puts a copy of the Vulkan loader into the bundle,
-making the bundle completely self-contained and self-referencing.
+#### Building with the Xcode Generator
 
-Note that the "install" target has a very different meaning compared to the Linux
-"make install" target.
-The Linux "install" copies the targets to system directories.
-In MacOS, "install" means fixing up application bundles.
-In both cases, the "install" target operations clean up the `RPATH`.
+To create and open an Xcode project:
 
-##### The Non-bundled vulkaninfo Application
+        mkdir build-xcode
+        cd build-xcode
+        cmake -GXcode ..
+        open Vulkan-Loader.xcodeproj
 
-There is also a non-bundled version of the `vulkaninfo` application that you can
-run from the command line:
+Within Xcode, you can select Debug or Release builds in the project's Build Settings.
+You can also select individual schemes for working with specific applications like `cube`.
```

Comment:
Not sure this line still applies.   The only app left in this repo is the test case.  If it actually works, then you might want to replace cube with it.